examples/langchain/README.md

```diff
@@ -89,4 +97,53 @@ query using the `marklogic_contextual_query_retriever.py` module in this project
 
 This retriever builds a term-query using words from the question. Then the term-query is
 added to the structured query and the merged query is used to select from the documents 
-loaded via `load_data.py`.
\ No newline at end of file
+loaded via `load_data.py`.
+
+## Testing using MarkLogic 12EA BM25 scoring and Vector functionality
```

Comment:
I would just call this "MarkLogic 12EA Vector Search". BM25 seems a lot less important, particularly as logtfidf may be better based on the dataset. It's nice that it's there, it just doesn't matter nearly as much as the fact that 12 has vector search.